### DISCUSSION TITLE
Multiple User access

### DISCUSSION THREAD

**Discussion by atc07** [teslamate-org/teslamate#984] [Ideas]
**What would you like to be added**: Ability for multiple users to be able to add their tesla account and login to the grafana dashboard to see their stats.

**Why is this needed**: In a shared community where multiple users are sharing the same charging station, it would be nice to only need to spin up 1 instance of teslamate that everyone can share. Users would use the access to submit their power consumption at the charging station for that location among the community. Access would be separated in their own profiles.

**Additional context**:

**Comment by matthieu-labas** [comment]
You can do that with the docker-compose solution: just make sure the ports are different for all Teslamate (default 3000), Postgres (default 5432) and Grafana (default 4000).

You could then probably improve it by creating a new database in the same Postgresql container, and different users in the same Grafana image, especially if you're running on small devices like Raspberry Pi.

**Comment by matthieu-labas** [comment]
Already discussed in #403

**Comment by JakobLichterfeld** [comment]
Unfortunately not supported by the API: https://github.com/teslamate-org/teslamate/discussions/963#discussioncomment-165631
